src/init/features/apphosting/index.ts

```diff
@@ -36,6 +39,105 @@ const apphostingPollerOptions: Omit<poller.OperationPollerOptions, "operationRes
   masterTimeout: 25 * 60 * 1_000,
   maxBackoff: 10_000,
 };
+/**
+ * Set up a new App Hosting backend.
+ */
+export async function doSetupWithDeveloperConnect(
```

Comment:
might be better to use the original `doSetup` function with a flag to use dev connect instead of cloud build